### PR TITLE
Bumps in garden : ign-math7

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/685

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@
 
 Build | Status
 -- | --
-Test coverage | [![codecov](https://codecov.io/gh/ignitionrobotics/ign-math/branch/ign-math6/graph/badge.svg)](https://codecov.io/gh/ignitionrobotics/ign-math)
-Ubuntu Focal | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_math-ci-ign-math6-focal-amd64)](https://build.osrfoundation.org/job/ignition_math-ci-ign-math6-focal-amd64)
-Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_math-ci-ign-math6-homebrew-amd64)](https://build.osrfoundation.org/job/ignition_math-ci-ign-math6-homebrew-amd64)
-Windows       | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_math-ci-ign-math6-windows7-amd64)](https://build.osrfoundation.org/job/ignition_math-ci-ign-math6-windows7-amd64)
+Test coverage | [![codecov](https://codecov.io/gh/ignitionrobotics/ign-math/branch/ign-math7/graph/badge.svg)](https://codecov.io/gh/ignitionrobotics/ign-math)
```
Comment (chapulina) [inline README.md:12]: Ahh here too we should have caught this earlier. It's all `main` for the badges

**Review by chapulina — APPROVED** [review]
LGTM with 🟢  CI and the links fixed

File: README.md (hunk 1 above)
Comment (methylDragon) [reply README.md:12]: It's alright, once all the PRs are in, I can do a sweeping regex find and replace

Comment (methylDragon) [reply README.md:12]: Just confirming that **every badge** everywhere is main?

Comment (chapulina) [reply README.md:12]: Every package's `main` branch should be using `main` links, yup

**Pull request merged**